diff --git a/node_modules/react-native-background-remover/ReactNativeBackgroundRemover.podspec b/node_modules/react-native-background-remover/ReactNativeBackgroundRemover.podspec
index 9479736..f9173ec 100644
--- a/node_modules/react-native-background-remover/ReactNativeBackgroundRemover.podspec
+++ b/node_modules/react-native-background-remover/ReactNativeBackgroundRemover.podspec
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => min_ios_version_supported }
+  s.platforms    = { :ios => "12.0" }
   s.source       = { :git => "https://github.com/atlj/react-native-background-remover.git", :tag => "#{s.version}" }
 
   s.source_files = "ios/**/*.{h,m,mm,swift}"
